### ISSUE TITLE
Support DynamoDB Transactions

### ISSUE THREAD

**Issue by jmarkbrooks** [Dynamoid/dynamoid#333]
https://aws.amazon.com/blogs/aws/new-amazon-dynamodb-transactions/

Does dynamoid support the new Transactions feature?

**Comment by andrykonchin** [comment]
No, Dynamoid doesn't support transaction yet. I hope it will be a part of the nearest release actually.

**Comment by andrykonchin** [comment]
Transactional modifying (with `TransactWriteItems`) is implemented in #688, #718, and #813.
